### PR TITLE
Update README files with new doc links and current product names

### PR DESCRIPTION
**Pull request by ancaemcken**
Addresses PBI 445594.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by CobyPear — APPROVED** [review]
Looks good!

File: packages/sitecore-jss-dev-tools/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# Sitecore JavaScript Services developer tools
+# Sitecore JavaScript Rendering SDK Developer Tools
 
-This module is provided as a part of Sitecore JavaScript Services. It contains developer tooling, such as development proxies and deployment scripts.
+This module is provided as a part of Sitecore JavaScript Rendering SDK (JSS). It contains developer tooling, such as development proxies and deployment scripts.
 
 Consult the [Sitecore JSS documentation](https://jss.sitecore.com) for further details.
```
Comment (ambrauer) [inline packages/sitecore-jss-dev-tools/README.md:5]: Missed a URL here?

**Review by ambrauer — CHANGES_REQUESTED** [review]
Is there any way we can avoid the add'l ongoing release maintenance (TODOs) introduced with these version-specific doc URLs? For example, use  https://doc.sitecore.com/xp/en/developers?
Otherwise looks great.

**Comment by ancaemcken** [comment]
@ambrauer Not at this time. I'm hoping we will soon have "latest" version tag :crossed_fingers:  In the meantime, I am more than happy to make the updates in these files (which is why I added the TODOs so I can quickly find the items that need updating). Hopefully, we will only have to update these once or twice before we get a "latest" version in the docs.

**Review by ambrauer — APPROVED** [review]

File: packages/sitecore-jss-dev-tools/README.md (hunk 1 above)
Comment (ancaemcken) [reply packages/sitecore-jss-dev-tools/README.md]: Oops :D

**Pull request merged**